### ISSUE TITLE
WGAN loss is right?

### ISSUE THREAD

**Issue by kikyou123** [shekkizh/WassersteinGAN.tensorflow#9]
I think your WGAN loss if not right, it should be:
self.discriminator_loss = tf.reduce_mean(logits_fake - logits_real)
self.gen_loss = tf.reduce_mean(-logits_fake)
ans your WGAN discriminator output shape is [batch_size, channel=1, height=2, width=2],it is a mistake?

**Comment by shekkizh** [comment]
The discriminator output is of shape [batch_size, 1]. The last layer of discriminator is a fc layer. 
The loss term has been previously discussed in issues #6 and #7 

Cheers.
